database_api_layer/controllers.py

```diff
@@ -366,3 +368,33 @@ def read_daily(self, session: Session, dailyObjectId: int):
       db.UserDailyObject.id == dailyObjectId
     ).order_by(db.UserDailyObject.scoreEarned.desc())
     return session.execute(query).all()
+class QuizController:
+  def read_one(self, session: Session,
+               quesId: Optional[int] = None):
+    query = select(db.DiscordQuiz)
+    if quesId is not None:
+      query = query.where(db.DiscordQuiz.id == quesId)
+    return session.scalar(query)
+
+  def read_quiz_answer(self, session: Session,
+                       quesId: Optional[int] = None):
+    query = select(db.DiscordQuizAnswer)
+    if quesId is not None:
+      query = query.where(db.DiscordQuizAnswer.discordQuizId == quesId)
+    return session.scalars(query).all()
+
+  def read_many(self, session: Session,
+                difficulty: Optional[str] = None,
+                category: Optional[str] = None):
+    query = select(db.DiscordQuiz)
+
+    sizeOfDb = session.scalar(select(func.count()).select_from(db.DiscordQuiz))
```

Comment:
Local variables should (mostly) be snake_case. Also, the name should be: `size_of_quiz_table` or `quiz_table_size`